src/OpenTelemetry.Instrumentation.SqlClient/README.md

```diff
@@ -60,45 +60,55 @@ For an ASP.NET application, adding instrumentation is typically done in the
+attribute is captured upon execution of a query.
 
-The following example shows how to use `SetStoredProcedureCommandName`.
+#### .NET Core - SetDbStatementForStoredProcedure and SetDbStatementForText
+
+On .NET Core, two properties are available: `SetDbStatementForStoredProcedure`
+and `SetDbStatementForText`. These properties control capturing of
+`CommandType.StoredProcedure` and `CommandType.Text` respectively.
+
+`SetDbStatementForStoredProcedure` is _true_ by default and will set
+[`db.statement`](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes)
+attribute to the stored procedure command name.
+
+`SetDbStatementForText` is _false_ by default (to prevent accidental capture of
+sensitive data that might be part of the SQL statement text). When set to
+`true`, the instrumentation will set [`db.statement`](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes)
+attribute to the text of the SQL command being executed.
+
+To disable capturing stored procedure commands use configuration like below.
 
 ```csharp
 using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .AddSqlClientInstrumentation(
-        options => options.SetStoredProcedureCommandName = false)
+        options => options.SetDbStatementForStoredProcedure = false)
     .AddConsoleExporter()
     .Build();
 ```
 
-### SetTextCommandContent (.NET Core)
-
-By default, when CommandType is CommandType.Text, this instrumentation will not
-set the `db.statement` attribute. This behavior can be enabled by setting
-`SetTextCommandContent` to true.
-
-The following example shows how to use `SetTextCommandContent`.
+To enable capturing of `sqlCommand.CommandText` for `CommandType.Text` use the
+following configuration.
 
 ```csharp
 using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .AddSqlClientInstrumentation(
-        options => options.SetTextCommandContent = true)
+        options => options.SetDbStatementForText = true)
     .AddConsoleExporter()
     .Build();
 ```
 
-## SetStatementText (.NET Framework)
+#### .NET Framework - SetDbStatement
 
-For .NET Framework, `SetTextCommandContent` and `SetStoredProcedureCommandName`
-are not available. Instead, `SetStatementText` should be used to control whether
-this instrumentation should set the `db.statement` attribute to the text of the
-`SqlCommand` being executed.
+For .NET Framework, `SetDbStatementForStoredProcedure` and
+`SetDbStatementForText` are not available. Instead, a single `SetDbStatement`
+property should be used to control whether this instrumentation should set the
+[`db.statement`](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#call-level-attributes)
+attribute to the text of the `SqlCommand` being executed.
```

Comment:
Should we clarify why .NET fx has only `SetDbStatement` and that it could be sp name or full query text itself?

Reply:
Thanks! Re-worded it a bit: tried to make it more clear right away that we're talking about both sp and full sql, plus added a bit of a background on differences.